digital_land/datatype/wkt.py

```diff
@@ -166,6 +169,11 @@ def normalise_geometry(geometry, simplification=0.000005):
     # ensure geometry is a MultiPolygon
     geometry = make_multipolygon(geometry)
 
+    if geometry:
+        if not geometry.is_valid:
+            issue = explain_validity(geometry)
```

Comment:
I'm assuming this is needed incase the make_polygon function creates an invalid geometry?

In which case I would suggest supplying the original issue instead of this issue

Reply:
Yes, to fix issues introduced by make_multipolygon

Function updated to return original issue👍